docs/api/overview.md

```diff
@@ -1,57 +1,79 @@
 ---
+title: Start developing
+sidebar_position: 1
 vega_network: TESTNET
 ---
+
 import Topic from '/docs/topics/_topic-development.mdx'
 import DataNodes from '@site/src/components/DataNodes';
-
-# Using the APIs
+import EthAddresses from '@site/src/components/EthAddresses';
 
 <Topic />
 
-There are a number of ways to interact with Vega through APIs and Ethereum bridge contracts, and below you can learn more about the reference documentation and tutorials that are currently available.
+This page describes how authentication and transactions work, and the versioning process.
+
+## Decentralised network
+The Vega protocol provides the backbone to form a network specifically built to support trading of financial products on markets proposed and voted on by members of the Vega community.
+
+Vega is a public, decentralised network run by independent validators. 
+
+There are a number of different servers that you'll need to interact while working with the network - data will generally be recieved from one service and transactions will be sent to another using data nodes. See the [data flow section on the architecture page](./architecture.md#data-flow).
+
+:::caution Supplying your own data
+Anyone that plans to build significant integrations, trading bots, or power backed services should expect to run their own data node, or collaborate with a data node provider, if one exists. 
+:::
+
+## Authentication
+Authentication takes several forms when interacting with Vega. When requesting data from a standard data node, such as through a query, you don't need to authenticate nor do you need an API token. Service providers offering free or paid data nodes may require you to authenticate, however. If this is the case you will need to refer to the provider's documentation for details of how to do so.
+
+If you want to submit transactions to the network, you'll need to ensure that you have a Vega Wallet, which provides authentication in the form of a signature when sending transactions. 
+
+When creating any scripts or software to interact with Vega, you'll need a wallet authentication token. 
```

Comment:
that's quite specific to the desktop wallet. I'm not sure what I'll change but probably write something differtnt

Reply:
If you want to write software to interact with the wallet, you'll need an API token whether or not you expect the user to use a desktop wallet... no?